src/FileUploader/FileUploader.js

```diff
@@ -0,0 +1,193 @@
+import { Box, IconButton, Typography } from "@mui/material";
+import React, { useEffect, useState } from "react";
+
+import DeleteIcon from "@mui/icons-material/Delete";
+import { DropzoneAreaBase } from "material-ui-dropzone";
+import FileUploadIcon from "@mui/icons-material/FileUpload";
+import PropTypes from "prop-types";
+import { makeStyles } from "@mui/styles";
+
+// file uploader component
+export default function FileUploader({
+  title = "Upload a File",
+  dropzoneText = "Drag & drop a file here or click",
+  filesLimit = 1,
+  multiple = false,
+  acceptedFiles = [],
+  maxFileSize = 3000000,
+  onAdd = () => {},
+  selectedFiles = []
+}) {
+  // files state
+  const [files, setFiles] = useState(selectedFiles);
+  // box text
+  const [uploaderText, setUploaderText] = useState(dropzoneText);
+
+  // styling
+  const useStyles = makeStyles(theme => ({
+    root: {
+      "& .MuiDropzoneArea-icon": {
+        color: "rgba(0, 0, 0, 0.60)",
+        display:
+          !multiple && files.length > 0
+            ? "none !important"
+            : "block !important",
+        fontSize: "22px !important",
+        height: "22px !important",
+        width: "22px !important"
+      },
+      "& .MuiDropzoneArea-text": {
+        color:
+          !multiple && files.length === 1 ? "#003063" : "rgba(0, 0, 0, 0.60)",
+        fontSize: "15px",
+        margin: "0 0 0 10px !important"
+      },
+      "& .MuiDropzoneArea-textContainer": {
+        display: "flex",
+        flexDirection: "row-reverse",
+        justifyContent: "center"
+      },
+      "& .MuiTypography-subtitle1": {
+        fontSize: "12px !important"
+      },
+      alignItems: "center",
+      borderWidth: "1px !important",
+      display: "flex",
+      justifyContent: "center",
+      marginBottom: "5px !important",
+      minHeight: "70px !important",
```

Comment:
Do we want variable height for the dropzone? @Sowbhagya-ipg

Reply:
Fixed height is fine.